Templating/FilterExtension.php

```diff
@@ -16,15 +16,13 @@
 
 class FilterExtension extends AbstractExtension
 {
-    use FilterTrait;
```

Comment:
This may be a BC since `FilterExtension` is not `@internal` and people may have extended it - a strange thing to do. But it's also the only way to prevent Symfony container to load the `CacheManager` when it's not necessary.

Reply:
with the current version of the PR, this is no BC break anymore, right?